files/zh-cn/web/javascript/reference/global_objects/date/index.html

```diff
@@ -131,8 +131,98 @@ <h3 id="实例属性">实例属性</h3>
+- {{jsxref("Date.prototype.setDate()")}}
+  - : 根据本地时间，设置一个 {{jsxref("Date")}} 对象在所属月份中的天数。
+- {{jsxref("Date.prototype.setFullYear()")}}
+  - : 根据本地时间，设置一个 {{jsxref("Date")}} 对象的完整年份（四位数年份）。
+- {{jsxref("Date.prototype.setHours()")}}
+  - : 根据本地时间，设置一个 {{jsxref("Date")}} 对象的小时数。
+- {{jsxref("Date.prototype.setMilliseconds()")}}
+  - : 根据本地时间，设置一个 {{jsxref("Date")}} 对象的豪秒数。
+- {{jsxref("Date.prototype.setMinutes()")}}
+  - : 根据本地时间，设置一个 {{jsxref("Date")}} 对象的分钟数。
+- {{jsxref("Date.prototype.setMonth()")}}
+  - : 根据本地时间，设置一个 {{jsxref("Date")}} 对象的月份。
+- {{jsxref("Date.prototype.setSeconds()")}}
+  - : 根据本地时间，设置一个 {{jsxref("Date")}} 对象的秒数。
+- {{jsxref("Date.prototype.setTime()")}}
+  - : 用一个从1970-1-1 00:00:00 UTC计时的毫秒数来为一个 {{jsxref("Date")}} 对象设置时间。用负数来设置更早的时间。
+- {{jsxref("Date.prototype.setUTCDate()")}}
+  - : 以协调世界时为标准，设置一个 {{jsxref("Date")}} 对象在所属月份中的天数。
+- {{jsxref("Date.prototype.setUTCFullYear()")}}
+  - : 以协调世界时为标准，设置一个 {{jsxref("Date")}} 对象的完整年份（四位数年份）。
+- {{jsxref("Date.prototype.setUTCHours()")}}
+  - : 以协调世界时为标准，设置一个 {{jsxref("Date")}} 对象的小时数。
+- {{jsxref("Date.prototype.setUTCMilliseconds()")}}
+  - : 以协调世界时为标准，设置一个 {{jsxref("Date")}} 对象的豪秒数。
+- {{jsxref("Date.prototype.setUTCMinutes()")}}
+  - : 以协调世界时为标准，设置一个 {{jsxref("Date")}} 对象的分钟数。
+- {{jsxref("Date.prototype.setUTCMonth()")}}
+  - : 以协调世界时为标准，设置一个 {{jsxref("Date")}} 对象的月份。
+- {{jsxref("Date.prototype.setUTCSeconds()")}}
+  - : 以协调世界时为标准，设置一个 {{jsxref("Date")}} 对象的秒数。
+- {{jsxref("Date.prototype.setYear()")}}
+  - : 根据本地时间，设置一个 {{jsxref("Date")}} 对象的相对年份（相对1900年，通常是 2 到 3 位数字）。请改用 {{jsxref("Date.setFullYear", "setFullYear")}} 。
+- {{jsxref("Date.prototype.toDateString()")}}
+  - : 以美式英语和人类易读的表述形式返回一个 {{jsxref("Date")}} 对象日期部分的字符串。
+- {{jsxref("Date.prototype.toISOString()")}}
+  - : 将指定 {{jsxref("Date")}} 对象转换成 ISO 格式表述的字符串并返回。
+- {{jsxref("Date.prototype.toJSON()")}}
+  - : 返回指定 {{jsxref("Date")}} 对象调用 {{jsxref("Date.prototype.toISOString()", "toISOString()")}} 方法的返回值。在 {{jsxref("JSON.stringify()")}} 中使用。
+- {{jsxref("Date.prototype.toGMTString()")}}
+  - : 基于 GMT (UTC) 时区来返回一个表述指定 {{jsxref("Date")}} 对象的字符串形式。请改用 {{jsxref("Date.prototype.toUTCString()", "toUTCString()")}}。
+- {{jsxref("Date.prototype.toLocaleDateString()")}}
+  - : 返回一个表述指定 {{jsxref("Date")}} 对象的日期部分字符串。该字符串格式因不同语言而不同。
+- {{jsxref("Date.prototype.toLocaleString()")}}
+  - : 返回一个表述指定 {{jsxref("Date")}} 对象的字符串。该字符串格式因不同语言而不同。
+- {{jsxref("Date.prototype.toLocaleTimeString()")}}
+  - : 返回一个表述指定 {{jsxref("Date")}} 对象时间部分的的字符串。该字符串格式因不同语言而不同。
+- {{jsxref("Date.prototype.toString()")}}
+  - : 返回一个字符串，表示该 {{jsxref("Date")}} 对象。覆盖了 {{jsxref("Object.prototype.toString()")}} 方法。
+- {{jsxref("Date.prototype.toTimeString()")}}
+  - : 以人类易读形式返回一个 {{jsxref("Date")}} 对象时间部分的字符串，该字符串以美式英语格式化。
+- {{jsxref("Date.prototype.toUTCString()")}}
+  - : 使用UTC时区，把一个 {{jsxref("Date")}} 对象转换为一个字符串。
+- {{jsxref("Date.prototype.valueOf()")}}
+  - : 返回一个 {{jsxref("Date")}} 对象的原始值。覆盖了 {{jsxref("Object.prototype.valueOf()")}} 方法。
```

Comment:
You have used markdown syntax in this part(in html file), which will not be rendered by [mdn/yari](https://github.com/mdn/yari). Please use definition table as follow:
https://github.com/mdn/translated-content/blob/77f8f285bb0c3352feca5b2a7f8331b1906ef0d5/files/zh-cn/web/javascript/reference/global_objects/date/index.html#L109-L120

Or you can convert this file to markdown like pr #5100

Reply:
@yin1999 I have updated the syntax